### PR TITLE
Wyvern status removal breaths should prioritize master and breath IDs are incorrect

### PR DESCRIPTION
**Pull request by rude-jerk**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by zach2good** [comment]
Hey! Please co-ordinate with @NTSabott to make sure you don't trample eachothers work :) 
https://github.com/project-topaz/topaz/pull/1258

**Comment by rude-jerk** [comment]
Waiting for https://github.com/project-topaz/topaz/pull/1258

**Comment by zach2good** [comment]
https://github.com/project-topaz/topaz/pull/1258 is now merged, which turns the hard-coded breath values into their tabled names - and also handles the +16 offset, so your PR can just become the changes to `doStatusBreath` 👍

File: scripts/globals/pets/wyvern.lua [hunk 1]
```diff
@@ -52,36 +52,46 @@ function doHealingBreath(player, threshold, breath)
     end
 end
 
+function doStatusBreath(target, player)
+    if target:hasStatusEffect(tpz.effect.POISON) then
+        player:getPet():useJobAbility(643, target)
```
Comment (zach2good) [inline scripts/globals/pets/wyvern.lua]: Can you break out all of these `player:getPet()`'s into a `local wyvern = player:getPet()` at the top please?

File: scripts/globals/pets/wyvern.lua [hunk 2]
```diff
@@ -52,36 +52,46 @@ function doHealingBreath(player, threshold, breath)
...
-                elseif member:hasStatusEffect(tpz.effect.PARALYSIS) and player:getPet():getMainLvl() > 40 then
-                    player:getPet():useJobAbility(629, member)
-                    break
-                elseif (member:hasStatusEffect(tpz.effect.CURSE_I) or member:hasStatusEffect(tpz.effect.DOOM)) and player:getPet():getMainLvl() > 60 then
-                    player:getPet():useJobAbility(637, member)
-                    break
-                elseif (member:hasStatusEffect(tpz.effect.DISEASE) or member:hasStatusEffect(tpz.effect.PLAGUE)) and player:getPet():getMainLvl() > 80 then
-                    player:getPet():useJobAbility(638, member)
-                    break
+            if not doStatusBreath(player, player) then
```
Comment (zach2good) [inline scripts/globals/pets/wyvern.lua:92]: This is a cool way of laying this out, I like it!

**Review by zach2good — APPROVED** [review]
All looks good, just make sure all of the effects are tabled when you resolve the conflicts :+1:

**Review by zach2good — APPROVED** [review]
Looks good, thanks! 👍

**Pull request merged**